src/livekit/useLiveKit.ts

```diff
@@ -39,9 +40,16 @@ import {
   ECConnectionState,
   useECConnectionState,
 } from "./useECConnectionState";
+import { MatrixKeyProvider } from "../e2ee/matrixKeyProvider";
+
+export enum E2EEMode {
```

Comment:
I'm confused why both this and `E2eeType` need to exist. Can we consolidate them?